framework/src/mesh/MooseMesh.C

```diff
@@ -489,6 +489,7 @@ MooseMesh::prepare(const MeshBase * const mesh_to_clone)
     const auto & add_sideset_ids = getParam<std::vector<BoundaryID>>("add_sideset_ids");
     _mesh_boundary_ids.insert(add_sideset_ids.begin(), add_sideset_ids.end());
     _mesh_sideset_ids.insert(add_sideset_ids.begin(), add_sideset_ids.end());
+    _mesh_nodeset_ids.insert(add_sideset_ids.begin(), add_sideset_ids.end());
```

Comment:
no the param is add_sideset_ids. 

let's rename the parameter to add_boundary_ids / add_boundary_names